keyboards/cosmo65/info.json

```diff
@@ -0,0 +1,83 @@
+{
+    "keyboard_name": "Cosmo65",
+    "manufacturer": "BentoBox Studio",
+    "url": "bentoboxstudio.xyz",
+    "maintainer": "BentoBoxStudio",
+    "usb": {
+        "vid": "0x7070",
+        "pid": "0x6331",
+        "device_version": "0.0.1"
+    },
+    "layouts": {
+        "LAYOUT_all": {
```

Comment:
if there are no other supported layouts here, this should be `LAYOUT`.  `LAYOUT_all` is generally for keyboards that have multiple layouts (the dz60's are a good example of that)

Reply:
I have changed them all, marking as resolved.